system-addon/content-src/components/TopSites/_TopSites.scss

```diff
@@ -129,7 +129,7 @@
       }
     }
 
-    .tippy-top-icon {
+    .rich-icon {
       position: absolute;
```

Comment:
Might be okay to just put a `background-color: white` here?

<img width="128" alt="image" src="https://user-images.githubusercontent.com/438537/30250736-a1ea1a54-9608-11e7-8afe-e6622e0253ae.png">

Or use `$background-primary` (grey-10 #f9f9fa):
<img width="125" alt="image" src="https://user-images.githubusercontent.com/438537/30250759-f7f70592-9608-11e7-81bd-48c6eb1925a7.png">

As long as it hides the fallback ;)
![screen shot 2017-09-10 at 9 07 46 am](https://user-images.githubusercontent.com/438537/30250763-07bcc246-9609-11e7-905a-92184fe0d011.png)

@aaronrbenson ?

Reply:
I lean towards the grey-10 option here just in case there's a white icon. It's not perfect but at least you could make out that something is there.